src/main/webapp/permissions-manage-files.xhtml

```diff
@@ -314,7 +321,9 @@
                                     <p:ajax event="toggleSelect"  update=":rolesPermissionsForm:selectionCounter" />
 
                                     <p:column selectionMode="multiple" style="width:2%;text-align:center"/>
-                                    <p:column headerText="#{bundle['dataverse.permissionsFiles.assignDialog.fileName']}">
+                                    <p:column width="98%"  headerText="#{bundle['dataverse.permissionsFiles.assignDialog.fileName']}">
```

Comment:
Not sure about the UX of the second column here for "Draft", while it may help when file has been deleted, it appears to be visual noise for all other use cases, by just have "Restricted" in two columns.

Reply:
The second column for Draft was not something I introduced here. I think it was done previously for the "same file name" issue that we talked about last week